docs/Operation-Architecture/Zenko_CLI/CRR_Metrics_and_Health.rst

```diff
@@ -0,0 +1,294 @@
+
+::
+
+    internalConnections: {
+        isrHealth: <ok || error>,
+        zookeeper: {
+            status: <ok || error>,
+            details: {
+                name: <value>,
+                code: <value>
+            }
+        },
+        kafkaProducer: {
+            Status: <ok || error>
+        }
+    }
+
+Status Code Responses
+---------------------
+
++----------+-------------------------------------------------------------------+
+| Response | Details                                                           |
++==========+===================================================================+
+| 200      | OK: success                                                       |
++----------+-------------------------------------------------------------------+
+| 403      | AccessDenied: request IP address does not have access             |
++----------+-------------------------------------------------------------------+
+| 404      | RouteNotFound: invalid route given                                |
++----------+-------------------------------------------------------------------+
+| 405      | MethodNotAllowed: the HTTP verb must be a GET                     |
++----------+-------------------------------------------------------------------+
+| 500      | InternalError: Can be caused by one of several internal components|
+|          | (i.e. the api server, Kafka, Zookeeper, or one of the Producers   |
+|          | for a topic)                                                      |
++----------+-------------------------------------------------------------------+
+
+Example Responses
+-----------------
+
+Healthcheck Request:
+
+::
+
+    [
+      {
+        "0":{
+          "nodeId":0,
+          "host":"server-node1",
+          "port":9092
+        },
+        ...
+        "4":{
+          "nodeId":4,
+          "host":"server-node5",
+          "port":9092
+        }
+      },
+      {
+        "metadata": {
+          "0":{
+            "topic":"backbeat-replication",
+            "partition":0,
+            "leader":4,
+            "replicas":[0,1,4],
+            "isr":[1,4,0]
+          },
+          ...
+          "4":{
+            "topic":"backbeat-replication",
+            "partition":4,
+            "leader":3,
+            "replicas":[0,3,4],
+            "isr":[4,3,0]
+          }
+        }
+      },
+      {
+        "internalConnections":{
+          "isrHealth":"ok",
+          "zookeeper":{
+            "status":"ok",
+            "details":{
+              "name":"SYNC_CONNECTED",
+              "code":3
+            }
+          },
+        "kafkaProducer":{
+          "status":"ok"
+          }
+        }
+      }
+    ]
+
+Backlog Request:
+
+::
+
+    "backlog":{
+      "description":"Number of incomplete replication operation (count) and number of incomplete MB transferred (size)",
+      "results":{
+        "count":4,
+        "size":"6.12"
```

Comment:
There shouldn't be a floating point value in the response. Can you confirm this @philipyoo?

Reply:
Changed Flare source. RST still out.